erddap_app/plots.py

```diff
@@ -47,7 +47,7 @@ def search_datasets(e, standard_name, cdm_data_type, min_time, max_time, skip_da
                 continue
 
     except HTTPError:
-        df = []
+        pass
```

Comment:
It this fails do we have a DataFrame to return below? Returning an empty list is not OK but maybe we should return an empty df or None. Not sure how this function is used down the pipeline to see if this makes sense.

Reply:
You were right. It was working in some instances, but I made new changes to the PR that make it more correct and more readable. Let me know what you think.